### PR TITLE
Review dip11

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
First commit wraps each line at ~80 characters as this makes editing markdown SOOOO much easier

All other commits should have descriptive enough commit messages. It started getting a bit late, so I feel like I didn't review the latter part as well as I did the beginning.

### CODE REVIEW

File: dip-0011.md [hunk 1]
```diff
@@ -175,49 +301,118 @@ Sending a contact request in Dashpay.
 </table>
 
 
-The master authentication key is the sole key can that sign Identity updates or disable the Identity.
+The master authentication key is the sole key can that sign Identity updates or
```
Comment (Cofresi) [inline dip-0011.md:304]:
```suggestion
The master authentication key is the sole key that can sign Identity updates or
```

**Review by Cofresi — CHANGES_REQUESTED** [review]
typo

**Review by shumkov — DISMISSED** [review]
👍

**Review by QuantumExplorer — APPROVED** [review]

File: dip-0011.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ in DIP0012.
 
 Cryptographic identities have long drawn substantial attention in the blockchain
 space. Various solutions and iterations have been proposed inside of Dash's
-development circle over the years. In this document we propose a novel approach
+development circle over the years. This document proposes a novel approach
```
Comment (QuantumExplorer) [inline dip-0011.md:55]: I actually prefer the first style more :)

File: dip-0011.md [hunk 3]
```diff
@@ -80,10 +80,10 @@ The Blockchain Users document (DIP0005) aimed to lay out a foundational
 transformation of how end users would interact with one another. The underlying
 Blockchain User transactions represented a starting point for this interaction.
 Blockchain Users would have then been discoverable by querying their associated
-usernames. At a stark difference from DIP0005, Identities are decoupled from
+usernames. A stark difference from DIP0005, Identities are decoupled from
```
Comment (QuantumExplorer) [inline dip-0011.md:83]: Not sure here @thephez ?

File: dip-0011.md [hunk 4]
```diff
@@ -68,11 +68,13 @@ that mining fees could be extremely variable. Registering an Identity would use
 more data than a standard transaction; if Dash blocks were full then the fees
 could be non-negligible and represent a barrier to entry into the system.
 
-Another big advantage is that the life cycle of an Identity exists directly
-inside the Platform consensus. If this was not the case then it would be very
+Another big advantage is that an Identity exists exclusively inside of 
+Platform's consensus. If this was not the case then it would be very
```
Comment (QuantumExplorer) [inline dip-0011.md:72]: The meaning here is that changes on an Identity exist inside the Platform consensus. I must say I prefer the first wording.

**Review by QuantumExplorer — CHANGES_REQUESTED** [review]

File: dip-0011.md [hunk 5]
```diff
@@ -134,7 +134,7 @@ Base58 is URL compatible and it is also less prone to spoofing.
 
 # Identity Life Cycle
 
-The life cycle of an Identity is built with multiple different underlying state
+The life cycle of an Identity is composed of multiple different underlying state
```
Comment (QuantumExplorer) [inline dip-0011.md:137]: I think the life cycle is a composition of states that are built with state transitions.

File: dip-0011.md [hunk 6]
```diff
@@ -166,21 +170,21 @@ Applications should encourage end users to maintain a healthy balance of
 credits for their Identities.
 
 As more applications come into existence, keys that are only used for a specific
-application, a group of applications (e.g., from a single developer) or to
+application, a group of applications (e.g. from a single developer) or to
```
Comment (thephez) [inline dip-0011.md:173]: Necessity of a comma here is debatable (dependent on which style guide one uses). If changing, it must be applied consistently through the doc.

File: dip-0011.md (hunk 3 above)
Comment (thephez) [reply dip-0011.md:83]: I'd tend towards the original. Probably would write something like "In stark contrast to DIP0005..." if I was writing it myself.

File: dip-0011.md (hunk 4 above)
Comment (shumkov) [reply dip-0011.md:72]: Agree!

File: dip-0011.md (hunk 6 above)
Comment (PastaPastaPasta) [reply dip-0011.md:173]: I don't think anywhere else in the doc has this kind of a comma

File: dip-0011.md (hunk 5 above)
Comment (PastaPastaPasta) [reply dip-0011.md:137]: @shumkov

File: dip-0011.md (hunk 6 above)
Comment (PastaPastaPasta) [reply dip-0011.md:173]: https://github.com/dashpay/dips/pull/66/commits/c7963e007017da552fe5cdbb3a55ae531fea7cdf

File: dip-0011.md (hunk 3 above)
Comment (PastaPastaPasta) [reply dip-0011.md]: https://github.com/dashpay/dips/pull/66/commits/095e6880cfdea9e4cb6c8100ef0682e1065e6123

File: dip-0011.md (hunk 4 above)
Comment (PastaPastaPasta) [reply dip-0011.md]: https://github.com/dashpay/dips/pull/66/commits/8ff7a01879fb513154325208be5b6384e1c0e9f7

File: dip-0011.md (hunk 2 above)
Comment (PastaPastaPasta) [reply dip-0011.md:55]: @thephez ?

File: dip-0011.md [hunk 7]
```diff
@@ -36,17 +36,57 @@
...
+documents. For example, if an Identity was disabled, but a state transition that
+the Identity had signed was submitted at roughly the same time. This would be
+a source of non-determinism, and while it could likely be mitigated, it is best
+to be avoided. 
+
+The Blockchain Users document (DIP0005) aimed to lay out a foundational
+transformation of how end users would interact with one another. The underlying
+Blockchain User transactions represented a starting point for this interaction.
+Blockchain Users would have then been discoverable by querying their associated
+usernames. In start contrast to DIP0005, Identities are decoupled from
```
Comment (thephez) [inline dip-0011.md:83]:
```suggestion
usernames. In stark contrast to DIP0005, Identities are decoupled from
```

File: dip-0011.md (hunk 3 above)
Comment (thephez) [reply dip-0011.md]: Okay other than the type :wink: Left a suggestion.

File: dip-0011.md (hunk 6 above)
Comment (thephez) [reply dip-0011.md:173]: :+1: Agree, I only see one other eg and it doesn't have one.

File: dip-0011.md (hunk 2 above)
Comment (thephez) [reply dip-0011.md:55]: Generally I don't have strong feelings on this one as there are varying opinions on which is better (plus it's not too unusual to see first person in BIPs). You guys can continue discussing if you like, but I will probably just merge whatever is here when all other items are resolved :slightly_smiling_face: 

BIP "we" count for reference:
``` sh
~/bips$ grep -Hrn 'we ' *.mediawiki | wc -l
219
```

File: dip-0011.md (hunk 5 above)
Comment (Cofresi) [reply dip-0011.md:137]: I agree with @QuantumExplorer

**Review by ka061600 — APPROVED** [review]

**Pull request closed without merging**